### DISCUSSION TITLE
add gccgo compilation support

### DISCUSSION THREAD

**Discussion by Nevereverest** [v2fly/v2ray-core#1542] [Ideas]
I build gccgo myself on linux system.Compiled by gccgo and the commands, it worked well.I think it would be a new alternative choice to use the package built by gccgo.(It has worked well yet which i used vmess+websocket)

**Comment by Nevereverest** [comment]
unfortunately, gccgo could only being built on linuc platform

**Comment by NickLiu635** [comment]
How to build by gccgo?
